content/security/index.adoc

```diff
@@ -8,70 +8,76 @@ The Jenkins project takes security seriously.
 
-We use the `jenkinsci-advisories` Google group to inform users about upcoming, scheduled security updates:
+Finally, the Jenkins project is a link:/security/cna[CVE Numbers Authority], and we submit CVE metadata simultaneously with the publication of security advisories, allowing automated security tools using CVE information to identify vulnerable installations.
 
-- link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[`jenkinsci-advisories` list archive]
-- link:https://feeds.feedburner.com/jenkins-security-advisories[`jenkinsci-advisories` RSS feed]
+[IMPORTANT]
+====
+Even if you run Jenkins on a private network and trust everyone in your team, security issues in Jenkins can still impact you:
 
-Whenever a security update is released, we inform the `jenkinsci-advisories` and link:https://oss-security.openwall.org/wiki/mailing-lists/oss-security[`oss-security`] mailing lists.
+- https://en.wikipedia.org/wiki/Cross-site_request_forgery[CSRF vulnerabilities] are a risk even if attackers have no direct access to Jenkins.
+- Does Jenkins build source code you haven't audited, using build scripts someone else wrote, displaying generated reports on its web UI? All of this is a potential a security concern.
+====
 
-IMPORTANT: We recommend you subscribe to link:https://groups.google.com/forum/#!forum/jenkinsci-advisories[`jenkinsci-advisories`] even if your Jenkins instance is not publicly accessible to receive timely notifications on security updates.
 
 [[reporting-vulnerabilities]]
 == How to Report a Security Vulnerability
 
 If you find a vulnerability in Jenkins, please report it in the issue tracker under the link:https://issues.jenkins-ci.org/browse/SECURITY[SECURITY project].
-This project is configured in such a way that only the reporter and the security team can see the details.
-By restricting access to this potentially sensitive information, we can work on a fix and deliver it before the method of attack becomes well-known.
+This project is configured in such a way that only the reporter, the maintainers, and the Jenkins security team can see the details.
+Restricting access to this potentially sensitive information allows core and plugin maintainers to develop effective security fixes that are safe to apply.
 We provide issue reporting guidelines and an overview of our process on link:reporting[Reporting Security Vulnerabilities].
 
 If you are unable to report using our issue tracker, you can also send your report to the private Jenkins Security Team mailing list:
 `jenkinsci-cert@googlegroups.com`
 
-To show our appreciation for your help, we'll send you link:https://wiki.jenkins.io/display/JENKINS/Rewards+for+reporting+security+issues[a small reward] for privately reported, valid vulnerability reports.
+To show our appreciation for your help, we'll send you https://wiki.jenkins.io/display/JENKINS/Rewards+for+reporting+security+issues[a small reward] for privately reported, valid vulnerability reports.
```

Comment:
Would be nice in the future to bring that page into the jenkins.io site as well.  Separate effort, not required before merging this page.

Reply:
Daniel was working on this, but decided to bound the effort and get what he had complete moving forward. I'm sure he'll get back to some of those other bits, including this one sometime.

Reply:
Yep, it's coming once this is merged. There's also another link to the wiki somewhere whose label weirdly now is the upcoming page title on jenkins.io 😅